src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -217,26 +216,35 @@ def excel2resources(
     return resources, True
 
 
-def _validate_excel_file(
-    classes_df: pd.DataFrame, df_dict: dict[str, pd.DataFrame]
-) -> ResourcesSheetsNotAsExpected | None:
-    for i, row in classes_df.iterrows():
-        index = int(str(i))  # index is a label/index/hashable, but we need an int
-        if not check_notna(row["super"]):
-            raise UserError(
-                f"Sheet 'classes' of 'resources.xlsx' has a missing value in row {index + 2}, column 'super'"
-            )
+def _prepare_classes_df(resource_dfs: dict[str, pd.DataFrame]) -> tuple[pd.DataFrame, dict[str, pd.DataFrame]]:
+    resource_dfs = {k.strip(): v for k, v in resource_dfs.items()}
+    if "classes" in resource_dfs:
```

Comment:
It is tiresome to first check for "classes", then for "Classes", ... and then, what's with "CLASSES"? I suggest doing that the other way round: On line 219, transform all keys to lowercase:
`resource_dfs = {k.strip().lower(): v for k, v in resource_dfs.items()}`

Reply:
I have thought about that, the problem with this is that we also change the all the other keys. When we compare the resource classes listed in the "classes" sheet to the sheets in the excel then we also need to change capitalisation for those. if there is a problem we will print out a message the names may not be identical to the ones used in the excel.